http/middleware.go

```diff
@@ -127,20 +148,20 @@ type LogData struct {
 	Panic              any
 }
 
-// WithData returns logger with filled fields.
-func WithData(l logger.ServerLogger, ld LogData) logger.ServerLogger {
+// WithData returns slog with filled fields.
+func WithData(l *slog.Logger, ld LogData) *slog.Logger {
 	l = l.With(
-		logger.Any("method", ld.Method),
-		logger.Any("path", ld.Path),
-		logger.Any("status_code", ld.ResponseStatusCode),
-		logger.Any("request_id", ld.RequestID),
-		logger.Any("duration_ms", ld.Duration.Milliseconds()),
+		slog.Any("method", ld.Method),
+		slog.String("path", ld.Path),
+		slog.Int("status_code", ld.ResponseStatusCode),
+		slog.String("request_id", ld.RequestID),
+		slog.Duration("duration_ms", ld.Duration),
 	)
 	if ld.Err != nil {
-		l = l.With(logger.Any("err", ld.Err.Error()))
+		l = l.With(slog.Any("error", ld.Err))
```

Comment:
If we want to use `error` instead of `err`, I will refactor it also in our template. `err` is everywhere at this moment.

Reply:
Thanks, very good point!